examples/memoize.py

```diff
@@ -12,8 +12,8 @@ def consume(value):
 
 # assumes you used `hera.set_global_token` and `hera.set_global_host` so that the workflow can be submitted
 with Workflow("memoize") as w:
-    g = Task("g", generate, outputs=[Parameter("out", value_from=ValueFrom(path="/out"))])
```

Comment:
@Trollgeir I used this as a test example for YAML export and noticed it was broken because the parameter names did not match, so adjusted it here